homeassistant/components/unifi/device_tracker.py

```diff
@@ -80,6 +81,22 @@
     controller = hass.data[unifi.DOMAIN][controller_id]
     tracked = {}
 
+    registry = await entity_registry.async_get_registry(hass)
+
+    # Restore clients that is not a part of active clients list.
+    for entity in registry.entities.values():
+
+        if entity.config_entry_id == config_entry.entry_id and \
+                entity.unique_id.startswith('dt-'):
```

Comment:
Is it enough to instead check the domain of the entity registry entry? It will be `'device_tracker'` for all device tracker entity entries.

Reply:
Sure, this way was only to keep with the theme of block and poe.